downstairs/build.rs

```diff
@@ -2,17 +2,19 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// Copyright 2022 Oxide Computer Company
+// Copyright 2023 Oxide Computer Company
 
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
 
-    if version_check::is_min_version("1.59").unwrap_or(false) {
-        println!("cargo:rustc-cfg=usdt_stable_asm");
+    if !version_check::is_min_version("1.59").unwrap_or(false) {
+        println!("cargo:rustc-cfg=usdt_need_asm");
```

Comment:
Is this print going to fire if the version is < 1.59?

Reply:
That's correct. We flipped the logic everywhere, including the meaning of the thing we print. The goal is to positively detect when we need to include `#![feature(asm)]`, which is for any version strictly before 1.59.